proxy/Cargo.toml

```diff
@@ -12,6 +12,7 @@ futures = "0.1"
 juniper = "0.13"
 juniper_warp = "0.4"
 log = "0.4"
+oscoin_client = { path = "/home/xla/dev/src/github.com/oscoin/oscoin-parity-wasm-prototype/client" }
```

Comment:
:eyes:

Reply:
@cloudhead Don't we all have a `/home/xla`. Needs some changes upstream, this is a draft and just wanted to commit before going to bed.